COPYING

```diff
@@ -39,245 +39,79 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
-     notice, this list of conditions and the following disclaimer in the
-     documentation and/or other materials provided with the distribution.
-  3. The name of the author may not be used to endorse or promote products
-     derived from this software without specific prior written permission.
+-------------------------------------------------------------------------------
+ OPTIONAL GPLv3 libraries
+-------------------------------------------------------------------------------
 
-  THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR
-  IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
-  OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
-  IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT,
-  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
-  NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
-  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
-  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
-  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
-  THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+Please be advised that the following libraries are covered under the GPLv3
+license.  If you choose to link CVC4 against one of these libraries, the
+resulting combined work is also covered under the GPLv3. If you want to make
+sure you build a version of CVC4 that uses no GPLed libraries, configure CVC4
+with the "--bsd" option before building (which is the default).  CVC4 can then
+be used in contexts where you want to license CVC4 under the (modified) BSD
+license.  See licenses/gpl-3.0.txt for more information.
 
-CVC4 by default links against The GNU Multiple Precision (GMP) Arithmetic
-Library, which is licensed under GNU LGPL v3. See the file GMP-LICENSE for a
-copy of that license.  Note that according to the terms of the LGPL, both
-CVC4's source, and the combined work (CVC4 linked with GMP) may (and do) remain
-under the more permissive CVC4 license.
-
-Alternatively, CVC4 can be optionally configured to link against CLN, the Class
-Library for Numbers, available here:
+CVC4 can be optionally configured to link against CLN, the Class Library for
+Numbers, available here:
 
   http://www.ginac.de/CLN/
 
-Please be advised that as this library is covered under the GPLv3, if you
-choose to use the combined work, "CVC4+CLN," by building CVC4 with CLN,
-then it is also covered under the GPLv3.  If you want to make sure you build
-a version of CVC4 that uses no GPLed libraries, configure CVC4 with the
-"--bsd" option before building (which is the default).  CVC4 can then be used
-in contexts where you want to license CVC4 under the (modified) BSD license.
-
 CVC4 can be optionally configured to link against GLPK, the GNU Linear
 Programming Kit, available here:
 
   http://www.gnu.org/software/glpk/
```

Comment:
CVC4 can be optionally configured to link against glpk-cut-log, a modified version of GLPK, the GNU Linear Programming Kit, available here:

http://github.com/timothy-king/glpk-cut-log